autodispose/src/main/java/com/uber/autodispose/KotlinExtensions.kt

```diff
@@ -253,3 +254,48 @@ inline fun <T> ParallelFlowable<T>.autoDisposable(provider: ScopeProvider): Para
 @CheckReturnValue
 inline fun <T> ParallelFlowable<T>.autoDispose(provider: ScopeProvider): ParallelFlowableSubscribeProxy<T> =
     this.`as`(AutoDispose.autoDisposable(provider))
+
+/** Executes a [body] with under an [AutoDisposeContext] backed by the given [scope]. */
```

Comment:
I guess you want `with` or `under`? i.e 
`Executes a [body] under an ....`
or
`Executes a [body] with an...`

Reply:
32e7e08